distribution/docs/src/main/resources/content/_running/external-ingest.adoc

```diff
@@ -0,0 +1,53 @@
+:title: External Methods of Ingesting Data
+:type: dataManagement
+:status: published
+:summary: External methods of ingesting data.
+:parent: Ingesting Data
+:order: 03
+
+== {title}
+
+Several third-party tools, such as https://curl.haxx.se/[cURL.exe] and the https://advancedrestclient.com/[Chrome Advanced Rest Client], can be used to send files and other types of data to ${branding} for ingest.
+
+.Windows Example
+----
+curl -H "Content-type: application/json;id=geojson" -i -X POST -d ${at-symbol}"C:\path\to\geojson_valid.json" ${secure_url}/services/catalog
+----
+
+.*NIX Example
+----
+curl -H "Content-type: application/json;id=geojson" -i -X POST -d ${at-symbol}geojson_valid.json ${secure_url}/services/catalog
+----
+
+Where: +
+*-H* adds an HTTP header. In this case, Content-type header `application/json;id=geojson` is added to match the data being sent in the request. +
+*-i* requests that HTTP headers are displayed in the response. +
+*-X* specifies the type of HTTP operation. For this example, it is necessary to POST (ingest) data to the server. +
+*-d* specifies the data sent in the POST request. The `${at-symbol}` character is necessary to specify that the data is a file. +
+
+The last parameter is the URL of the server that will receive the data.
+
+This should return a response similar to the following (the actual catalog ID in the id and Location URL fields will be different):
+
+.Sample Response
+[source,http,linenums]
+----
+HTTP/1.1 201 Created
+Content-Length: 0
+Date: Mon, 22 Apr 2015 22:02:22 GMT
+id: 44dc84da101c4f9d9f751e38d9c4d97b
+Location: ${secure_url}/services/catalog/44dc84da101c4f9d9f751e38d9c4d97b
+Server: Jetty(7.5.4.v20111024)
+----
+
+. Verify the entry was successfully ingested by entering in a browser the URL returned in the POST response's HTTP header. For instance in our example, it was `/services/catalog/44dc84da101c4f9d9f751e38d9c4d97b`. This should display the catalog entry in XML within the browser.
+. Verify the catalog entry exists by executing a query via the OpenSearch endpoint.
+. Enter the following URL in a browser /services/catalog/query?q=ddf. A single result, in Atom format, should be returned.
+
+
+A resource can also be ingested with metacard metadata associated with it using the multipart/mixed content type.
```

Comment:
I do not know what this means. I think it means that if you something like an image or PDF file, AND you have an XML representation of a metacard, then ingest them both in one go and they will be associated with each other.

Reply:
That's the assumption I have about it as well.